lang/java/reef-common/src/main/java/org/apache/reef/client/DriverConfiguration.java

```diff
@@ -177,6 +179,8 @@
    */
   public static final OptionalImpl<EventHandler<ContextMessage>> ON_CONTEXT_MESSAGE = new OptionalImpl<>();
 
+  // ***** MISC
```

Comment:
This is to separate the bindings from `// ***** CONTEXT HANDLER BINDINGS`, since these parameters don't belong with ContextHandlers.